tests/phpunit/tests/locale.php

```diff
@@ -140,4 +140,27 @@ public function test_is_rtl() {
 		$this->locale->text_direction = 'ltr';
 		$this->assertFalse( $this->locale->is_rtl() );
 	}
+
+	/**
+	 * @covers WP_Locale::get_word_count_type
```

Comment:
Can you please add ticket number?

Reply:
@costdev added it to the new refreshed https://github.com/WordPress/wordpress-develop/pull/4019/